settings/checkstyle/checkstyle.xml

```diff
@@ -17,7 +17,9 @@
     <property name="fileExtensions" value="java"/>
   </module>
 
-  <module name="NewlineAtEndOfFile"/>
+  <module name="NewlineAtEndOfFile">
+    <property name="lineSeparator" value="lf"/>
```

Comment:
On windows, it actually should end with CRLF if git was configured correctly - https://help.github.com/articles/dealing-with-line-endings/#platform-windows Could we execute the following command before cloning? 

```
git config --global core.autocrlf true
```